src/System.Management.Automation/engine/hostifaces/PowerShellProcessInstance.cs

```diff
@@ -41,7 +41,7 @@ static PowerShellProcessInstance()
         }
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-linux`
Invoke-WebRequest tests.Denial of service.Charset Parsing
```powershell
Expected the actual value to be greater than 10, but got 0.207703495927987.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1: line 1925
1925:             $pathologicalRatio | Should -BeGreaterThan 10
```